concent_api/utils/client_auth.py

```diff
@@ -0,0 +1,42 @@
+from functools                      import wraps
+
+from django.http                    import JsonResponse
+
+from golem_messages.exceptions      import FieldError
+from golem_messages.exceptions      import MessageError
+from golem_messages.exceptions      import MessageFromFutureError
+from golem_messages.exceptions      import MessageTooOldError
+from golem_messages.exceptions      import TimestampError
+
+from core.validation                import validate_golem_message_signed_with_key
+from core.exceptions                import Http400
+
+from utils.shortcuts                import load_without_public_key
+
+
+def client_auth(message_handler):
```

Comment:
This variable should be still called `view`, because it should not matter if we are wrapping just view or view wrapped in some other decorator. I think it should be good to write some tests that check this.

Reply:
The name is little bit misleading - it actually does more with loading golem message and handling its exceptions. Please call this one `require_golem_message` or something more descriptive.